src/commands/projects.ts

```diff
@@ -169,26 +169,36 @@ const list = async (props: CommonProps & { orgId?: string }) => {
     return result;
   };
 
-  const ownedProjects = getList(props.apiClient.listProjects);
+  const ownedProjects = await getList(props.apiClient.listProjects);
   const sharedProjects = props.orgId
-    ? undefined
-    : getList(props.apiClient.listSharedProjects);
+    ? []
+    : await getList(props.apiClient.listSharedProjects);
 
   const out = writer(props);
 
-  out.write(await ownedProjects, {
-    fields: PROJECT_FIELDS,
-    title: 'Projects',
-  });
+  if (ownedProjects.length) {
+    out.write(ownedProjects, {
+      fields: PROJECT_FIELDS,
+      title: 'Projects',
+    });
+  } else {
+    out.text(
+      "\n\nYou don't have any projects yet. See how to create a new project:\n",
+    );
+    out.text('> neonctl projects create --help\n\n');
+  }
 
-  if (sharedProjects) {
-    out.write(await sharedProjects, {
+  if (sharedProjects.length) {
+    out.write(sharedProjects, {
       fields: PROJECT_FIELDS,
       title: 'Shared with you',
     });
   }
-
   out.end();
+
+  if (!sharedProjects.length && !props.orgId) {
+    out.text('\nNo projects have been shared with you\n');
+  }
```

Comment:
`out.write()` and `out.text()` behave differently, causing a different output order. Perhaps we should come up with a better abstraction for printing text (especially, with formatting options)

Reply:
I think we just need to update `text` method to push to same chunks array as `write` does

Reply:
Do you want my help with fixing it's behavior?

Reply:
I agree!

>Do you want my help with fixing it's behavior?

That should be straightforward, let me try first :)

Reply:
@duskpoet I think I figured out a better abstraction for adding optional messages to tables in case data is empty  [`edf5a5c` (#284)](https://github.com/neondatabase/neonctl/pull/284/commits/edf5a5cb72357e509c5134129078d6bbe542c694)

What do you think?